pyigm/field/igmfield.py

```diff
@@ -0,0 +1,236 @@
+
+        Returns
+        -------
+
+        rho : Quantity (array usually)
+          Impact parameter(s) in kpc
+        """
+        if los_coord is None:
+            los_coord = self.coord
+        # Coord
+        o_coord = SkyCoord(ra=obj['RA']*u.deg, dec=obj['DEC']*u.deg)
+        ang_sep = o_coord.separation(los_coord).to('arcmin')
+        # Cosmology
+        kpc_amin = self.cosmo.kpc_comoving_per_arcmin(obj['Z'] ) # kpc per arcmin
+        rho = ang_sep * kpc_amin / (1+obj['Z']) # Physical
+        # Return
+        return rho
+
+    def get_associated_galaxies(self, z, los_coord=None, R=300*u.kpc, dv_tol=500*u.km/u.s):
+        """Return a Table of associated galaxies for a given redshift and separation
+
+        Parameters
+        ----------
+        z : float
+          Redshift for association (usually IGM/CGM absorber redshift)
+        R : Quantity
+          Radius of impact parameter for association [300kpc]
+        dv_tol : Quantity
+          Velocity window for association [500km/s]
+        los_coord : SkyCoord, optional
+          Line-of-sight coordinates
+
+        Returns
+        -------
+        assoc_gal : Table
+          Table of associated galaxies (if any)
+        rho : Quantity
+          Impact parameters [kpc]
+        """
+        # los_radec
+        if los_coord is None:
+            los_coord = self.coord
+        # Cut on z first
+        dv_gal = const.c.to('km/s') * (self.galaxies['Z']-z)/(1+z)  # Approximate
+        gdz = np.where(np.abs(dv_gal)<dv_tol)[0]
+        if len(gdz) == 0:
+            return None
+        #
+        gdz_gal = self.galaxies[gdz]
+        rho = self.calc_rhoimpact(gdz_gal, los_coord)  # Could add this to Table
+        #
+        gd_rho = np.where(rho < R)[0]
+        if len(gd_rho) == 0:
+            return None
+        # Return
+        return gdz_gal[gd_rho], rho[gd_rho]
+
+    def get_observed(self, theta, subtab=None):
+        """Generate a Table of observed targets within an angular offset of field center
+
+        Parameters
+        ----------
+        theta : Quantity or Angle
+          Angular radius
+
+        Returns
+        -------
+        obs_targ : Table
+          Sub-table of targets that have been observed within this radius
+        obs_dates : List
+          List of observing dates [eventually might add to Table]
+        indices : array
+          Indices from the main table
+        """
+        if (self.targets is None) or (self.observing is None):
+            raise ValueError('IgmGalaxyField: Need to fill the target and/or observing table first!')
+        if subtab is None:
+            # Trim on angular cut first
+            targ_coord = SkyCoord(ra=self.targets['TARG_RA']*u.deg,
+                dec=self.targets['TARG_DEC']*u.deg)
+            sep = self.coord.separation(targ_coord)
+            gdsep = np.where(sep < theta)[0]
+            if len(gdsep) == 0:
+                return None
+            # Set all to False to start
```

Comment:
Do not understand the inline comment in this context. The line below does not set them to False, does it?